config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show] do
```

Comment:
- [ ] Kindly include the routes for the `new` and `create` actions in the `config/routes.rb` file to enable the creation of posts.

  Example:
  ```rb
    resources :posts, only: %i[index show new create] do
  ```